src/react_native/utils.cljs

```diff
@@ -37,3 +40,26 @@
         props       (when (map? first-child) first-child)
         children    (if props (rest argv) argv)]
     [props children]))
+
+(defn convert-prop-value
```

Comment:
btw im not sure why is it in react-native.utils it should be in utils/something

Reply:
react-native shouldn't know about reagent, so probably it should be utils/reagent